compiler/rustc_codegen_llvm/src/asm.rs

```diff
@@ -212,6 +213,10 @@ impl AsmBuilderMethods<'tcx> for Builder<'a, 'll, 'tcx> {
                             }
                         }
                     } else {
+                        if let Some(label) = get_llvm_label_from_str(s) {
```

Comment:
A template string piece may consist of multiple lines (e.g. raw string literals). These strings could define more than one label.

Reply:
Ah thanks, I didn't think of this case